front/src/components/store/MonacoEditor/MonacoEditor.vue

```diff
@@ -330,14 +330,14 @@ export default {
     handleNewError (newError: FeathersError) {
       if (this.editor) {
         const currentEditorModel = this.editor.getModel()
-        const { data: { location }, message = '' } = newError
-        if (currentEditorModel && location && message) {
+        const { data = { }, message = '' } = newError
```

Comment:
Did you try to put `any` here instead of `{ }` or maybe a record `Record<string, any>` ?

Reply:
I just set a default value to `data`, sometimes it's `undefined`.
I don't know how to type a property of a variable.